docs/creation/supply-chain.md

```diff
@@ -19,13 +19,14 @@ actor TempWriter
 
   new create(auth: FileAuth, file_name: String) =>
     let dir = FilePath.mkdtemp(auth)?
+    let log = FilePath.from(dir, file_name)?
     _file = File(dir)
```

Comment:
```suggestion
    _file = File(log)
```

Reply:
I see this change is made below in the next iteration, but do not see a reason it should be different here for the purpose of learning.